packages/shorebird_cli/lib/src/commands/patch/patcher.dart

```diff
@@ -67,7 +68,9 @@ https://docs.shorebird.dev/status#link-percentage-ios
   Future<void> assertPreconditions();
 
   /// Asserts that the combination arguments passed to the command are valid.
-  Future<void> assertArgsAreValid() async {}
+  Future<void> assertArgsAreValid() async {
+    argResults.assertAbsentOrValidKeyPair();
+  }
```

Comment:
I might even move this up to the `PatchCommand` layer so Patchers don't need to remember to call `super` (just literally moving `argResults.assertAbsentOrValidKeyPair();` to PatchCommand's `createPatch` method)